modules/tests/shared/src/test/scala/lucuma/core/math/UnitsSuite.scala

```diff
@@ -3,10 +3,12 @@
 
 package lucuma.core.math
 
-import coulomb._
-import coulomb.mks._
-import coulomb.si.Meter
-import lucuma.core.math.units._
+import coulomb.{*, given}
+import coulomb.policy.standard.given
+import coulomb.policy.spire.standard.given
+import coulomb.units.mks.{*, given}
+import coulomb.units.si.{*, given}
+import lucuma.core.math.units.{*, given}
```

Comment:
@erikerlandson so then these should all be fine, right?

Reply:
yes, those look right, EXCEPT that if you import `coulomb.policy.spire.standard.given` you should not also import `coulomb.policy.standard.given` - these are one case where they do not "union"

Also, it should be harmless but you only need to `import coulomb.*`

Reply:
Thanks, I just pushed a commit with those changes but I still get the same error 😕

Reply:
:sob: 

at the moment I don't see what is wrong, although I have never tried using type aliases like `PicoMeter` in a definition - I would have assumed it should work but it might be worth trying `Pico * Meter` directly in the definition

Reply:
if that is a problem I might need to make more use of `tr.dealias`

Reply:
ugh.  I'll experiment with more `dealias` to see if that helps